### PR TITLE
Add match average rank

### PR DESCRIPTION
**Pull request by builder-247**
* To be used in player overview page

This should work right? It is kinda hard to test the scanner stuff - if I understood it correctly all new matches originate from scanner

### CODE REVIEW

File: sql/create_tables.cql [hunk 1]
```diff
@@ -122,6 +122,7 @@ CREATE TABLE IF NOT EXISTS player_matches (
   firstblood_claimed int,
   party_size int,
   net_worth int,
+  average_rank int,
```
Comment (howardchung) [inline sql/create_tables.cql:125]: don't think it's needed here

File: sql/create_tables.sql [hunk 2]
```diff
@@ -135,7 +135,8 @@ CREATE TABLE IF NOT EXISTS player_matches (
   roshans_killed int,
   observers_placed int,
   party_size int,
-  net_worth int
+  net_worth int,
+  average_rank int
```
Comment (howardchung) [inline sql/create_tables.sql:139]: also not needed here since this is pro matches only

File: store/queries.js [hunk 3]
```diff
@@ -485,7 +485,7 @@ function getMatchRankTier(match, cb) {
     }
     // Remove undefined/null values
     const filt = result.filter(r => r);
-    const avg = Math.floor(filt.map(r => Number(r)).reduce((a, b) => a + b, 0) / filt.length);
+    const avg = Math.floor(filt.map(r => Number(r)).reduce((a, b) => a + b, 0) / filt.length) || null;
```
Comment (howardchung) [inline store/queries.js:488]: when will this be null?

Comment (builder-247) [reply store/queries.js:488]: If none of the players have rank data

**Comment by builder-247** [comment]
Should I also remove the `skill` field from the API response?

**Review by howardchung — APPROVED** [review]

**Pull request merged**